src/components/shared/page.tsx

```diff
@@ -82,13 +82,18 @@ interface BodyProps {
     capi: Content;
 }
 
+interface ElementWithScript {
```

Comment:
Should this be `ElementWithResources`, and have an array instead of one script?
That opens the door to prefetch more than one thing